streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -2210,6 +2210,9 @@ public void shouldComputeOffsetSumForStandbyTask() throws Exception {
         restoringTask.setChangelogOffsets(changelogOffsets);
 
         assertThat(taskManager.getTaskOffsetSums(), is(expectedOffsetSums));
+
+        final Set<TopicPartition> assignment = singleton(new TopicPartition("assignment", 0));
+        Mockito.verify(mockitoConsumer).resume(assignment);
```

Comment:
I do not think we need this verification. Method `resume()` is called in some setup code of the test.

Reply:
I was trying to keep the same strength of verification as was already present from the method I was trying to get rid of i.e.
```
    private static void expectRestoreToBeCompleted(final Consumer<byte[], byte[]> consumer) {
        final Set<TopicPartition> assignment = singleton(new TopicPartition("assignment", 0));
        expect(consumer.assignment()).andReturn(assignment);
        consumer.resume(assignment); <-- THIS
        expectLastCall();
    }
```
If you think I can relax this, I am happy to remove it from the tests where you have made the same remark

Reply:
With EasyMock you need to call `verify()` to verify a call. I do not see a call to `verify()` in the old code. That tells me that verifying the call to `consumer.resume()` was not relevant in the old code of this test. Additionally, the goal of this test, is to verify the result of `taskManager.getTaskOffsetSums()`. For that you need some tasks assigned. So part of the setup of this test is to install an assignment so that you get tasks assigned. With `Mockito.verify(mockitoConsumer).resume(assignment)` you would test rather an unimportant part of the setup of this test.
To sum up, I have the impression that you verify more in the migration that it was verified in the old code and you added a rather unimportant verification.